### PR TITLE
Adding updated OWNERS file

### PR DESCRIPTION
**Pull request by fbm3307**
Adding updated OWNERS file as its required by prow

Similar PRs:

- Host-Operator - https://github.com/codeready-toolchain/host-operator/pull/1029
- Member-Operator - https://github.com/codeready-toolchain/member-operator/pull/567
- Registration-Service - https://github.com/codeready-toolchain/registration-service/pull/430

### CODE REVIEW

**Review by mfrancisc — APPROVED** [review]

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- alexeykazakov
+- MatousJobanek
+- michaelkleinhenz
```
Comment (alexeykazakov) [inline OWNERS:6]:
```suggestion
```

**Review by alexeykazakov — APPROVED** [review]

**Comment by fbm3307** [comment]
e2e is stuck testing it again
/test e2e

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- sbryzak
+- xcoulon
+- rajivnathan
+- ranakan19
+- mfrancisc
+
+reviewers:
+- alexeykazakov
+- MatousJobanek
+- michaelkleinhenz
```
Comment (xcoulon) [inline OWNERS:15]: I'm not sure if @michaelkleinhenz still wants review our PRs? ;)

**Review by xcoulon — APPROVED** [review]

File: OWNERS (hunk 2 above)
Comment (fbm3307) [reply OWNERS:15]: he is added as CODEOWNER, so was trying to make OWNER  file same as it

Comment (fbm3307) [reply OWNERS:15]: a miss,  he is not there as codeowner.. so should we remove his name or keep his name here ?

**Pull request merged**